src/xrAICore/Navigation/graph_edge.h

```diff
@@ -30,7 +30,7 @@ class CEdgeBase
 };
 
 template <typename _edge_weight_type, typename _vertex_type, typename _edge_data_type>
-class CEdge : public CEdgeBase<_edge_weight_type, _vertex_type>
+class CEdgeT : public CEdgeBase<_edge_weight_type, _vertex_type>
```

Comment:
It's conflicting with some class with the same name?

Reply:
It`s conflict with this: `typedef CVertex<_data_type, _vertex_id_type, CGraphAbstract> CVertex`

Reply:
It is better to rename the typedef, not the class!

Reply:
It`s was do less changes in code

Reply:
Maybe, then rename it to CGraphEdge? It seems better than CEdgeT because causes questions about what `T` actually means.

Reply:
But, however, renaming the typedef is still better even if it will cause much code changes.

Reply:
`edge_t` is the option here :)